lib/src/components/pagination/pagination.helper.ts

```diff
@@ -0,0 +1,53 @@
+export const numOfPaginationItemsToRender = (
```

Comment:
Based on the name of the function I was expecting a `number` as result instead of a `number[]`.

Might be good to update the name to something like:
```suggestion
export const getPaginationItemsToRender = (
```

Reply:
yeah good point